pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -249,15 +284,24 @@ func convertToPermission(rule *rbacproto.AccessRule) *policyproto.Permission {
 	return &policyproto.Permission{Rule: rules}
 }
 
-// convertToPrincipals converts a list of subjects to principals.
-func convertToPrincipals(bindings []*rbacproto.ServiceRoleBinding) []*policyproto.Principal {
-	principals := make([]*policyproto.Principal, 0)
+// convertToPrincipals converts lists of subjects to principals(enforced/permissive mode).
```

Comment:
What is the definition of enforced/permissive principals. If the binding rules is Enforced, but bind to role that is Permissive, what is the principals mode then?

Reply:
updated comments, 'mode' should be only used to servicerole/binding now.